code/web/src/modules/survey/Survey/List.js

```diff
@@ -59,15 +61,31 @@ class List extends PureComponent {
               this.props.surveyItems.isLoading
                 ? <Loading/>
                 : this.props.surveyItems.surveyItems.length > 0
-                  ? this.props.surveyItems.surveyItems.map(item => (
-                    <div key={item.id} style={{ margin: '2em', float: 'left' }}>
+                  ? this.props.surveyItems.surveyItems.map((item, i) => (
+                    <div
+                      key={'item#'+i+'survey#'+this.props.match.params.page}
```

Comment:
I am not 100% sure what this does 
key={'item#'+i+'survey#'+this.props.match.params.page}

Reply:
React wants unique keys for Components in an Array. This now has a `key` of the shape

```Javascript
"item#1survey#1"
```